Kernel/Output/HTML/Templates/Standard/ProcessManagement/ActivityDialogHeader.tt

```diff
@@ -69,7 +69,7 @@
             <input type="hidden" name="IsMainWindow" id="IsMainWindow" value="[% Data.IsMainWindow | html %]"/>
             <input type="hidden" name="IsProcessEnroll" id="IsProcessEnroll" value="[% Data.IsProcessEnroll | html %]"/>
             <input type="hidden" name="FormID" value="[% Data.FormID | html %]"/>
-            <div class="LayoutPopup ARIARoleMain">
+            <div class="LayoutPopup ARIARoleMain NoMarginTop">
```

Comment:
Please use the already existing class `no-margin`.  This sets all margins to zero. Since only the top is set, this would be a solid solution.

[var/httpd/htdocs/skins/Agent/default/css/Core.FlexboxModel.css](https://github.com/znuny/Znuny/blob/dev/var/httpd/htdocs/skins/Agent/default/css/Core.FlexboxModel.css#L163)